plugins/colorschemes/base16/default.nix

```diff
@@ -37,94 +42,147 @@ helpers.vim-plugin.mkVimPlugin config {
       ])
     ];
 
+  settingsExample = {
+    telescope_borders = true;
+    indentblankline = false;
+    dapui = false;
+  };
+
+  settingsOptions = with helpers.defaultNullOpts; {
+    telescope = mkBool true ''
+      Whether to enable telescope integration.
+    '';
+    telescope_borders = mkBool false ''
+      Whether to enable themed borders in the telescope integration.
+    '';
+    indentblankline = mkBool true ''
+      Whether to enable indentblankline integration.
+    '';
+    notify = mkBool true ''
+      Whether to enable notify integration.
+    '';
+    ts_rainbow = mkBool true ''
+      Whether to enable ts_rainbow integration.
+    '';
+    cmp = mkBool true ''
+      Whether to enable cmp integration.
+    '';
+    illuminate = mkBool true ''
+      Whether to enable illuminate integration.
+    '';
+    lsp_semantic = mkBool true ''
+      Whether to enable lsp_semantic integration.
+    '';
+    mini_completion = mkBool true ''
+      Whether to enable mini_completion integration.
+    '';
+    dapui = mkBool true ''
+      Whether to enable dapui integration.
+    '';
+  };
+
   extraOptions = {
     colorscheme =
       let
+        names = pipe "0123456789ABCDEF" [
+          stringToCharacters
+          (map (i: "base0" + i))
+        ];
```

Comment:
As nice as this is, it's made totally redundant by declaring `examples` in the let block :sob: 

We can remove `names` and then `mapAttrs` the example instead

Reply:
Ok ^^
Marking this as solved then.